reference/resources/felixconfig.md

```diff
@@ -102,7 +102,7 @@ spec:
 | wireguardMTU                       | MTU set on the WireGuard interface created by Felix. Zero value means auto-detect. See [Configuring MTU]({{ site.baseurl }}/networking/mtu). | int | int | 0 |
 | wireguardRoutingRulePriority       | WireGuard routing rule priority value set up by Felix. If you change the default value, set it to a value most appropriate to routing rules for your nodes. | 1-32765 | int | 99 |
 | xdpRefreshInterval                 | Period at which Felix re-checks the XDP state in the dataplane to ensure that no other process has accidentally broken {{site.prodname}}'s rules. Set to 0 to disable XDP refresh. | `5s`, `10s`, `1m` etc. | duration | `90s` |
-| xdpEnabled                         | Enable XDP acceleration for host endpoint policies. [Default: `true`] | true,false | boolean | `true` |
+| xdpEnabled                         | When `bpfEnabled` is `false`: enable XDP acceleration for host endpoint policies.  When `bpfEnabled` is `true` this setting has no effect.  [Default: `true`] | true,false | boolean | `true` |
```

Comment:
> When `bpfEnabled` is `true` this setting has no effect

Do we need to make it explicitly clear that "no-effect" means "always on"?

Reply:
Well "always on" doesn't sound quite right to me.  The reality is more like a combination of
- "in BPF mode, the implementation of policy is always accelerated, using the best available BPF technology"
- "Calico v3.20 BPF mode now implements untracked ingress Deny policy, whereas previously BPF mode did not implement any untracked policy"

Can you think of wording for that that is concise enough to go here, and also precisely correct?

Reply:
Perhaps for this table we put:
```
When `bpfEnabled` is `true` this setting has no effect: in BPF mode, the implementation of policy is always accelerated, using the best available BPF technology.
```

while we put both those statements in the explanatory part of the doc (line 124ish)

Reply:
That's nice, I'll add it.

Reply:
Oh sorry, I didn't read your comment carefully enough.  But I don't think the statement about untracked policy being newly (and partially) supported belongs here.  And I think the shorter "has no effect" statement is better for the table.

Reply:
Perhaps:
```suggestion
| xdpEnabled                         | When `bpfEnabled` is `false`: enable XDP acceleration for host endpoint policies.  When `bpfEnabled` is `true` this setting has no effect and is treated as always `true`.  [Default: `true`] | true,false | boolean | `true` |
```